src/Mono.Android/metadata

```diff
@@ -1659,7 +1659,47 @@
   <!-- The setter 'setColor (long)' can't be merged with the existing 'virtual long Color { get; }' because it's a breaking change -->
   <attr api-since="29" path="/api/package[@name='android.graphics']/class[@name='Paint']/method[@name='setColor' and count(parameter)=1 and parameter[1][@type='long']]" name="propertyName"></attr>
 
-  <!--
+  <!-- API-33 Adjustments -->
+  <!-- Work around a generator bug where having multiple Listener methods with the same name cause multiple conflicting EventArgs classes to be created. -->
+  <!-- This is the "default" method version, so it's ok to remove it from an interface. -->
+  <remove-node api-since="33" path="/api/package[@name='android.accessibilityservice']/interface[@name='AccessibilityService.MagnificationController.OnMagnificationChangedListener']/method[@name='onMagnificationChanged' and count(parameter)=3 and parameter[1][@type='android.accessibilityservice.AccessibilityService.MagnificationController'] and parameter[2][@type='android.graphics.Region'] and parameter[3][@type='android.accessibilityservice.MagnificationConfig']]" />
+
+  <!-- Fix covariant return type -->
+  <attr api-since="33" path="/api/package[@name='java.security.cert']/class[@name='URICertStoreParameters']/method[@name='clone' and count(parameter)=0]" name="managedReturn">Java.Lang.Object</attr>
+
+  <!-- We have 2 methods with the same name and the same parameter name. One needs to be enumified but not the other.
+       Our methodmap.csv isn't capable of only choosing 1 of the methods, so it sets them both as the enum, making both have the
+       same signature. So we are going to set the enum transformation here instead, where we can specify a path that only affects a single method. -->
+  <attr path="/api/package[@name='android.content.pm']/class[@name='PackageManager']/method[@name='getActivityInfo' and count(parameter)=2 and parameter[1][@type='android.content.ComponentName'] and parameter[2][@type='int']]/parameter[@name='flags']" name="enumType">Android.Content.PM.PackageInfoFlags</attr>
```

Comment:
Reading things like this, I wonder if we could instead start using `//class/method/@jni-signature`, a'la:

```xml
<attr
    path="/api/package[@name='android.content.pm']/class[@name='PackageManager']/method[
      @name='getActivityInfo' and
      starts-with(@jni-signature, '(Landroid/content/ComponentName;I)')
    ]"
    name="enumType"
>Android.Content.PM.PackageInfoFlags</attr>
```

It would certainly be *shorter*…